src/lib/util/debug.c

```diff
@@ -162,6 +162,8 @@ char const CC_HINT(used) *__lsan_default_suppressions(void)
 		"leak:libSystem_atfork_child"
 #elif defined(__linux__)
 		"leak:kqueue"
+#else
```

Comment:
to be safe, I would change this to `#elif defined(__FreeBSD__)` so that the change fixes it only for FreeBSD.

i.e. make the smallest change which fixes the problem.

Reply:
Got it. make sense. done